frontend/public/style/_overrides.scss

```diff
@@ -399,3 +399,8 @@ ul.pf-v5-c-tree-view__list {
   color: var(--pf-global--Color--100);
   background-color: var(--pf-global--palette--black-600);
 }
+
+// PF5 Popover component applies an new inline width:auto style that overrides it's own pf-v5-c-popover min-width: var(--pf-v5-c-popover--MinWidth). So the pf-v5-c-popover container is not a consistent size and can resize when additional content is loaded which causes the alignment to be incorrect.
```

Comment:
Is there an upstream bug we can include a link to?

Reply:
Their isn't yet. I relayed my findings to @nicolethoen on the PatternFly team who was going to do some more investigations.